examples/Features/Steno/Steno.ino

```diff
@@ -61,7 +61,7 @@ KEYMAPS(
 KALEIDOSCOPE_INIT_PLUGINS(GeminiPR);
 
 void setup() {
-  KeyboardHardware.serialPort().begin(9600);
+  Kaleidoscope.serialPort().begin(9600);
```

Comment:
It kinda is. But it's important to have it here - and elsewhere -, because the serialport setup in `Kaleidoscope_::setup()` is a _workaround_. Once we figure out how to remove that, we will, and the sketches will have to set up serial on their own. So we want to keep pretending that the workaround is not there, to not teach bad habits.